script/glam/backfill_glean

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# example:
```

Comment:
Script name is missing? Or is that intentional?

Reply:
Actually the script name starts after the environment variables